scancodeio/urls.py

```diff
@@ -47,10 +47,11 @@
     path("accounts/profile/", AccountProfileView.as_view(), name="account_profile"),
 ]
 
+
 urlpatterns = auth_urlpatterns + [
     path("admin/", admin.site.urls),
     path("api/", include(api_router.urls)),
-    path("license/", include((licenses.urls, "license_app"))),
```

Comment:
No reasons to rename the initial "license".

Reply:
ok